src/translations/es.json

```diff
@@ -0,0 +1,37 @@
+{
+  "estado": {
```

Comment:
You shouldn't translate keys. Translations wouldn't work in that case. Please, translate only values.